build/linux/installer/scripts/livenessprobe.sh

```diff
@@ -49,29 +49,31 @@ fi
 
 
 #optionally test to exit non zero value if fluentd is not running
-#fluentd not used in sidecar container
-if [ "${CONTAINER_TYPE}" != "PrometheusSidecar" ]  && [ "${GENEVA_LOGS_INTEGRATION_SERVICE_MODE}" != "true" ]; then
-  (ps -ef | grep "fluentd" | grep -v "grep")
-  if [ $? -ne 0 ]
-  then
-   echo "fluentd is not running" > /dev/termination-log
-   exit 1
-  fi
-  # fluentd launches by default supervisor and worker process
-  # so adding the liveness checks individually to handle scenario if any of the process dies
-  # supervisor process
-  (ps -ef | grep "fluentd" | grep "supervisor" | grep -v "grep")
-  if [ $? -ne 0 ]
-  then
-   echo "fluentd supervisor is not running" > /dev/termination-log
-   exit 1
-  fi
-  # worker process
-  (ps -ef | grep "fluentd" | grep -v "supervisor" | grep -v "grep" )
-  if [ $? -ne 0 ]
-  then
-   echo "fluentd worker is not running" > /dev/termination-log
-   exit 1
+if [ "$AZMON_RESOURCE_OPTIMIZATION_ENABLED" != "true" ]; then
```

Comment:
we would need fluentd for replicaset scenarios, are you setting the AZMON_RESOURCE_OPTIMIZATION_ENABLED only for DS ? If yes, I didnt see that in the code. can you please double check on this.

Reply:
will add check in tomlparser-agent-config.rb